apmpackage/apm/agent/input/template.yml.hbs

```diff
@@ -65,14 +62,11 @@ apm-server:
         {{#each tls_curve_types}}
           - {{this}}
         {{/each}}
-    {{/if}}
     write_timeout: {{write_timeout}}
-    {{#if tail_sampling_policies}}
     sampling:
       tail:
         enabled: true
```

Comment:
won't this cause tail based sampling to always be enabled? we should probably infer the value on whether the `tail_sampling_policies` has any policies.

Reply:
+

Reply:
updated, thanks!